src/components/Breadcrumb.vue

```diff
@@ -70,6 +71,7 @@ export default {
           }
           breadcrumb.unshift({ type: 'root' });
         }
+        breadcrumb.push(...append);
```

Comment:
Why was this change made?  Won't this cause a request to fire to the same endpoint for every keystroke?  As the user types the name of the new folder, the result is now padded in time by a network request.

Reply:
Oh, good catch. @subdavis 
The reason for changing this is becuase Vuetify has been console.warn that `[Vuetify] 'default slot' is deprecated, use ':items and scoped slot "item"' instead`, but when using the scoped slot 'item', the `appends` has to be added to the `items`, but I didn't realize it takes a toll on the performance if simply add like this. 
Good news is simply extract adding the appends in a computed of its own would solve the problem. I'll fix this and leave a note for why having an additional computed.